protoc-gen-grpc-gateway/descriptor/services.go

```diff
@@ -49,7 +53,7 @@ func (r *Registry) loadServices(file *File) error {
 	return nil
 }
 
-func (r *Registry) newMethod(svc *Service, md *descriptor.MethodDescriptorProto, opts *options.HttpRule) (*Method, error) {
+func (r *Registry) newMethod(svc *Service, md *descriptor.MethodDescriptorProto, optss []*options.HttpRule) (*Method, error) {
```

Comment:
s/optss/options/

Let's be explicit that we are talking about plural of opts instead of it being a typo of opts.